src/java.desktop/share/classes/javax/swing/plaf/basic/BasicLookAndFeel.java

```diff
@@ -459,6 +459,9 @@ private void initResourceBundle(UIDefaults table) {
     /**
      * Populates {@code table} with the defaults for the basic look and
      * feel.
+     * <p>
+     * The default button margin value is (2, 14, 2, 14), which may
+     * greatly differ from other LookAndFeel defaults.
```

Comment:
This will need a CSR, which is fine.
But
(1) Basic is an abstract L&F .. it can't be the place to say what Metal does since most L&Fs derive from Basic
And something about a JButton ought to be closer to ButtonUI or JButton.
(2) Do we have precedent for calling out the exact number of (user space) pixels ?
I'd prefer some more waffly wording like 
"The default margins may vary greatly depending on the L&F".

Reply:
For (1), I've considered some alternative spots for this note. MetalButtonUI and MetalLookAndFeel for example. I decided to put it in BasicLookAndFeel since the default value is set here, but I see your point. I can add an additional note in the doc for, say, MetalButtonUI if preferred.

In (2), you're right that there's no explicit pixel count for anything. I'll update this to be more general.

Reply:
I have removed the doc change from this PR to focus mainly on fixing the regression first. Could you re-review when you get the chance?